lib/api/index.js

```diff
@@ -1,4 +1,18 @@
+'use strict'
+
+/**
+ * Handles a request to a not implemented `/api/` endpoint.
+ * @param req
+ * @param res
+ * @param next
+ */
+function notImplemented (req, res, next) {
+  res.status(404).send('API not implemented')
```

Comment:
Ping regarding content type of the 404 response. Would be nice if this didn't just trigger the default HTML not found page since this is for an API endpoint.